books/free-programming-books-langs.md

```diff
@@ -2222,6 +2222,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 * [A community-driven Ruby style guide](https://github.com/bbatsov/ruby-style-guide)
 * [Core Ruby Tools](https://launchschool.com/books/core_ruby_tools) - Launch School (HTML)
+* [Design Patterns in Ruby](https://github.com/abdelrahman-hsn/ruby-ebooks/blob/master/Design%20Patterns%20in%20Ruby%2C%20Addison-Wesley%20(2008).pdf) - Addison-Wesley
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.